### PR TITLE
feat (community): Started integrating BE with threads home page

### PR DESCRIPTION
**Pull request by chef-danny-d**

### CODE REVIEW

File: components/common/community/threads/thread/thread.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import * as React from 'react'
 import { GoArrowUp, GoCommentDiscussion } from 'react-icons/go'
 import { TbShare } from 'react-icons/tb'
-import { Anchor } from '../../../links/anchor/anchor'
+import { Anchor } from '@/common/links/anchor/anchor'
```
Comment (joeldesante) [inline components/common/community/threads/thread/thread.tsx:4]: Please use NextJS Link tag instead

File: components/common/pages/layouts/layout/layout.tsx [hunk 2]
```diff
@@ -19,8 +20,8 @@ export const Layout = ({ children }) => {
 	}
 
 	// TODO: we might want to uncomment this once we have next routing integrated
-	// if (status !== 'loading' && session === null) {
```
Comment (joeldesante) [inline components/common/pages/layouts/layout/layout.tsx:22]: delete tis plz!

File: components/common/pages/module_list/module_list.tsx [hunk 3]
```diff
@@ -7,13 +7,13 @@ export const ModuleList = ({ modules }: ModuleListProps) => {
 			{modules.map((module, moduleIndex) => (
 				<Anchor
```
Comment (joeldesante) [inline components/common/pages/module_list/module_list.tsx:8]: No Anchor taaaag

Comment (chef-danny-d) [reply components/common/pages/module_list/module_list.tsx:8]: Update to use Link

File: components/common/pages/layouts/layout/layout.tsx (hunk 2 above)
Comment (chef-danny-d) [reply components/common/pages/layouts/layout/layout.tsx:22]: Removed comments

File: components/common/community/threads/thread/thread.tsx (hunk 1 above)
Comment (chef-danny-d) [reply components/common/community/threads/thread/thread.tsx]: Updated to use Next Link

**Pull request merged**